lib/cc/analyzer/engine.rb

```diff
@@ -34,7 +34,7 @@ def run(stdout_io, container_listener)
         )
 
         container.on_output("\0") do |raw_output|
-          CLI.debug "#{name} engine output: #{raw_output.inspect}"
+          CLI.debug("#{name} engine output: #{raw_output.gsub("\n", "\\n").gsub("\t", "\\t")}")
```

Comment:
We could simply not escape it: I'm not escaping it in stderr below right now.

The only downside of that, for me, is it can be less obvious where engine output begins or ends when it has leading or trailing whitespace.